infrastructure/main.tf

```diff
@@ -13,9 +13,14 @@ locals {
   nonPreviewVaultName = "${var.raw_product}-${var.env}"
   vaultName           = "${local.is_preview ? local.previewVaultName : local.nonPreviewVaultName}"
   local_env           = "${local.is_preview ? "aat" : var.env}"
+  local_ase = "${(var.env == "preview" || var.env == "spreview") ? (var.env == "preview" ) ? "core-compute-aat" : "core-compute-saat" : local.ase_name}"
 
   sku_size = "${var.env == "prod" || var.env == "sprod" || var.env == "aat" ? "I2" : "I1"}"
 
+  ccdCnpUrl = "http://ccd-data-store-api-${local.local_env}.service.${local.local_ase}.internal"
+  ccdApiUrl = "${var.env == "sprod" || var.env == "demo" ? local.ccdCnpUrl : "false"}"
```

Comment:
this config is only enabling it in sprod and demo you want to enable it everywhere,
so just delete this line and rename the var one line up to be ccdUrl

Reply:
I copied this of the case main.tf ... not sure about which is where tbh.
will do that